test/deps/binaryen/binaryen_test.cpp

```diff
@@ -56,17 +57,18 @@ TEST(BinaryenTest, Example1) {
   auto fun_impl = [&](int a) { ASSERT_EQ(a, expected_argument); };
 
   // wast code with imported function's call
-  auto fmt = boost::format("(module\n"
-                           " (type $v (func))\n"
-                           " (import \"%1%\" \"%2%\" (func $%2% (param i32)))\n"
-                           " (start $starter)\n"
-                           " (func $starter (; 1 ;) (type $v)\n"
-                           "  (call $%2%\n"
-                           "   (i32.const %3%)\n"
-                           "  )\n"
-                           " )\n"
-                           ")") %
-             env_name % fun_name % expected_argument;
+  auto fmt = boost::format(
+                 "(module\n"
+                 " (type $v (func))\n"
+                 " (import \"%1%\" \"%2%\" (func $%2% (param i32)))\n"
+                 " (start $starter)\n"
+                 " (func $starter (; 1 ;) (type $v)\n"
```

Comment:
Is `starter` some kind of well-known entry point name like `main`?

Reply:
Look like that.